http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -99,3 +105,10 @@ docker run -d --restart always --network host --name welder \
 --env STAGING_BUCKET=$WELDER_STAGING_BUCKET \
 --env SHOULD_BACKGROUND_SYNC="false" \
 $WELDER_WELDER_DOCKER_IMAGE
+
+echo "{ 'env': { 'WORKSPACE_ID': '${WORKSPACE_ID}', 'WORKSPACE_STORAGE_CONTAINER_ID': '${WORKSPACE_STORAGE_CONTAINER_ID}', 'WORKSPACE_NAME': '${WORKSPACE_NAME}', 'WORKSPACE_STORAGE_CONTAINER_URL': '${WORKSPACE_STORAGE_CONTAINER_URL}' }}" | jq . >> wsenv.json
+jq -s . /usr/local/share/jupyter/kernels/julia-1.6/kernel.json wsenv.json > /usr/local/share/jupyter/kernels/julia-1.6/kernel.json
```

Comment:
Was testing and this puts `env` in a sub object which isn't correct. Existing:
```
jupyter@vm0186808a1:~$ jq -s . /usr/local/share/jupyter/kernels/julia-1.6/kernel.json wsenv.json[  {
    "display_name": "Julia 1.6.0",
    "argv": [
      "/opt/julia/julia-1.6.0/bin/julia",
      "-i",
      "--color=yes",
      "--project=@.",
      "/opt/julia/latest/packages/packages/IJulia/AQu2H/src/kernel.jl",
      "{connection_file}"
    ],
    "language": "julia",
    "env": {},
    "interrupt_mode": "signal"
  },
  {
    "env": {
      "WORKSPACE_ID": "foo",
      "WORKSPACE_STORAGE_CONTAINER_ID": "bar",
      "WORKSPACE_NAME": "name",
      "WORKSPACE_STORAGE_CONTAINER_URL": "url"
    }
  }
]
```

Using `-s add` seems to work. Fixed:
```
jupyter@vm0186808a1:~$ jq -s add /usr/local/share/jupyter/kernels/julia-1.6/kernel.json wsenv.json
{
  "display_name": "Julia 1.6.0",
  "argv": [
    "/opt/julia/julia-1.6.0/bin/julia",
    "-i",
    "--color=yes",
    "--project=@.",
    "/opt/julia/latest/packages/packages/IJulia/AQu2H/src/kernel.jl",
    "{connection_file}"
  ],
  "language": "julia",
  "env": {
    "WORKSPACE_ID": "foo",
    "WORKSPACE_STORAGE_CONTAINER_ID": "bar",
    "WORKSPACE_NAME": "name",
    "WORKSPACE_STORAGE_CONTAINER_URL": "url"
  },
  "interrupt_mode": "signal"
}
```